### PR TITLE
[SOAR-1894] Brute Force - Domain Account

### PR DESCRIPTION
**Pull request by mhofert-r7**
## Requirements

Workflow contributors, verify you have completed the following items by checking them off:

### Checklist
- [ ] Workflow verified as working in a test
- [ ] Workflow bundle includes `Name_of_Workflow.icon` file, `help.md` file, `workflow.spec.yaml` file, and `screenshots` directory
- [ ] Workflow title should follow general [Title of Articles](http://grammar.yourdictionary.com/capitalization/rules-for-capitalization-in-titles.html) format
- [ ] Workflow includes a detailed Description
- [ ] Workflow step names follow [Title of Articles](http://grammar.yourdictionary.com/capitalization/rules-for-capitalization-in-titles.html) format
- [ ] Triggers, Decisions, and other workflow steps include descriptions where needed
- [ ] Workflow includes at least one `Final Report` artifact that summarizes workflow execution details
- [ ] Workflow bundle (directory) is named after the title of the workflow in snake case with first letters capitalized, e.g. `workflows/Name_of_Workflow/`
- [ ] `help.md` file includes Description, Key Features, Requirements, Setup, Technical Details, Version History, and References
- [ ] `workflow.spec.yaml` file includes `products`, `name`, `title`, `description`, `use_cases` tags, `keywords` tags, `source_url`, and the `name` for each screenshot matches a PNG image in the `screenshots` directory

### Assessment

Post your results with screenshots in this PR using the following style:

```
### Screenshots

Screenshots help your reviewers provide valuable feedback on your contribution. To add a screenshot here, simply drag and drop the PNG files from a file explorer.

#### Workflow Builder

<Insert Workflow Image Here>

#### Job

<Insert Job Image Here>

#### Artifact

<Insert Artifact Image Here>
```

### CODE REVIEW

File: workflows/Block_Domain_Account_with_Active_Directory_from_InsightIDR/help.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+## Setup
+
+Import the workflow from the Rapid7 Extension Library and proceed through the Import Workflow wizard in InsightConnect. Import plugins, create or select connections, and rename the workflow as a part of the Import Workflow wizard as necessary.
+
+Once the workflow has been imported, **It is recommended that you add you add the appropriate emails to the `Block Domain Account Decision` step so that the appropriate people are alerted that action needs to be taken**
+
+Activate the workflow in order to trigger it.
+
+TODO IDR Set UP
```
Comment (tschmidtke-r7) [inline workflows/Block_Domain_Account_with_Active_Directory_from_InsightIDR/help.md:25]: Do we want to add details here before release?

File: workflows/Block_Domain_Account_with_Active_Directory_from_InsightIDR/help.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+# Usage
+
+
```
Comment (jschipp-r7) [inline workflows/Block_Domain_Account_with_Active_Directory_from_InsightIDR/help.md:29]:
```suggestion

```

File: workflows/Block_Domain_Account_with_Active_Directory_from_InsightIDR/help.md (hunk 1 above)
Comment (jschipp-r7) [inline workflows/Block_Domain_Account_with_Active_Directory_from_InsightIDR/help.md:25]:
```suggestion
```

File: workflows/Block_Domain_Account_with_Active_Directory_from_InsightIDR/help.md [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+# Documentation
+
+## Setup
+
+Import the workflow from the Rapid7 Extension Library and proceed through the Import Workflow wizard in InsightConnect. Import plugins, create or select connections, and rename the workflow as a part of the Import Workflow wizard as necessary.
+
+Once the workflow has been imported, **It is recommended that you add you add the appropriate emails to the `Block Domain Account Decision` step so that the appropriate people are alerted that action needs to be taken**
+
+Activate the workflow in order to trigger it.
```
Comment (jschipp-r7) [inline workflows/Block_Domain_Account_with_Active_Directory_from_InsightIDR/help.md:23]:
```suggestion
After activation, this workflow is triggered when an InsightIDR Brute Force - Domain Account UBA alert occurs.
```

File: workflows/Block_Domain_Account_with_Active_Directory_from_InsightIDR/help.md [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+## Setup
+
+Import the workflow from the Rapid7 Extension Library and proceed through the Import Workflow wizard in InsightConnect. Import plugins, create or select connections, and rename the workflow as a part of the Import Workflow wizard as necessary.
+
+Once the workflow has been imported, **It is recommended that you add you add the appropriate emails to the `Block Domain Account Decision` step so that the appropriate people are alerted that action needs to be taken**
+
+After activation, this workflow is triggered when an InsightIDR Brute Force - Domain Account UBA alert occurs.
+
+
+# Usage
```
Comment (jschipp-r7) [inline workflows/Block_Domain_Account_with_Active_Directory_from_InsightIDR/help.md:26]:
```suggestion
```

File: workflows/Block_Domain_Account_with_Active_Directory_from_InsightIDR/help.md [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+
```
Comment (jschipp-r7) [inline workflows/Block_Domain_Account_with_Active_Directory_from_InsightIDR/help.md:26]:
```suggestion
```

File: workflows/Block_Domain_Account_with_Active_Directory_from_InsightIDR/help.md [hunk 6]
```diff
@@ -0,0 +1,50 @@
...
+## Setup
+
+Import the workflow from the Rapid7 Extension Library and proceed through the Import Workflow wizard in InsightConnect. Import plugins, create or select connections, and rename the workflow as a part of the Import Workflow wizard as necessary.
+
+Once the workflow has been imported, **It is recommended that you add you add the appropriate emails to the `Block Domain Account Decision` step so that the appropriate people are alerted that action needs to be taken**
+
+After activation, this workflow is triggered when an InsightIDR Brute Force - Domain Account UBA alert occurs.
+
+
```
Comment (jschipp-r7) [inline workflows/Block_Domain_Account_with_Active_Directory_from_InsightIDR/help.md:25]:
```suggestion
```

File: workflows/Block_Domain_Account_with_Active_Directory_from_InsightIDR/help.md [hunk 7]
```diff
@@ -0,0 +1,50 @@
...
+The Workflow will run when a InsightIDR Brute Force - Domain Account UBA alert is triggered.
```
Comment (jschipp-r7) [inline workflows/Block_Domain_Account_with_Active_Directory_from_InsightIDR/help.md:25]:
```suggestion
```

File: workflows/Block_Domain_Account_with_Active_Directory_from_InsightIDR/help.md [hunk 8]
```diff
@@ -0,0 +1,48 @@
...
+# Documentation
+
+## Setup
+
+Import the workflow from the Rapid7 Extension Library and proceed through the Import Workflow wizard in InsightConnect. Import plugins, create or select connections, and rename the workflow as a part of the Import Workflow wizard as necessary.
+
+Once the workflow has been imported, **It is recommended that you add you add the appropriate emails to the `Block Domain Account Decision` step so that the appropriate people are alerted that action needs to be taken**
+
+After activation, this workflow is triggered when an InsightIDR Brute Force - Domain Account UBA alert occurs.
+
```
Comment (jschipp-r7) [inline workflows/Block_Domain_Account_with_Active_Directory_from_InsightIDR/help.md:24]:
```suggestion
```

**Review by jschipp-r7 — APPROVED** [review]

**Pull request merged**